server/packets.py

```diff
@@ -410,35 +410,40 @@ def write_logout_packet(user_id: int) -> bytes:
 
 # SPECTATOR_JOINED = 13
 
+
 def write_spectator_joined_packet(user_id: int) -> bytes:
     return write_packet(
-        packet_id=ClientPackets.START_SPECTATING,
+        packet_id=ServerPackets.SPECTATOR_JOINED,
```

Comment:
all `write_*` functions will always return server packets. the server cannot send client packets.